automation_testing/tests/Components/dataview/save_as.spec.ts

```diff
@@ -1,9 +1,9 @@
 import { test, expect } from "@playwright/test";
-import { dataview_data, saveAs_data } from "../../utils/data/dataview_data";
-import { DataviewPage } from "../../pages/DataView/DataViewPage";
-import { SaveAsComponent } from "../../pages/DataView/SaveAsComponent";
-import { PaginationComponent } from "../../pages/DataView/PaginationComponent";
-import { ColumnsComponent } from "../../pages/DataView/ColumnsComponent";
+import { dataview_data, saveAs_data } from "../../../utils/data/dataview_data";
```

Comment:
I found that these files that are marked in red need to update some import paths. 

![image](https://user-images.githubusercontent.com/87880265/179999089-f3509619-61ab-46c6-a6fe-1b1ca7ff9fc5.png)

![image](https://user-images.githubusercontent.com/87880265/179999213-0e8d5308-9c2d-4fce-a9f5-e813744b6800.png)